pandas/tests/plotting/test_converter.py

```diff
@@ -207,15 +207,15 @@ def test_conversion_float(self):
 
         rs = self.dtc.convert(Timestamp("2012-1-1 01:02:03", tz="UTC"), None, None)
         xp = converter.dates.date2num(Timestamp("2012-1-1 01:02:03", tz="UTC"))
-        tm.assert_almost_equal(rs, xp, decimals)
+        tm.assert_almost_equal(rs, xp, check_less_precise=decimals)
```

Comment:
can't you change this to use atol/rtol? otherwise you will get deprecation warnings